.cypress/integration/ad/dashboard/ad_dashboard.spec.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/// <reference types="cypress" />
```

Comment:
Can we remove this line?

Reply:
This is needed to enable `Intelligent Code Completion` in IDE: https://docs.cypress.io/guides/tooling/IDE-integration.html#Triple-slash-directives